guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -256,6 +256,140 @@ ifeval::["{build}" == "satellite"]
 When the installation completes, the host also registers to {ProjectServer} using the activation key and installs the necessary configuration and management tools from the *{project-client-name}* repository.
 endif::[]
 
+[id="creating-hosts-with-uefi-http-boot-provisioning_{context}"]
+=== Creating Hosts with UEFI HTTP Boot Provisioning
+
+You can provision hosts from {Project} using the UEFI HTTP Boot.
+This is the only method with which you can provision hosts in IPv6 network.
```

Comment:
Are you saying that UEFI HTTP Boot is not supported in IPv4?

Reply:
> We should be more clear here. "UEFI HTTP Boot is fully tested on IPv6 and known to work on IPv4."

I asked Radovan and he said that the UEFI HTTP Boot provisioning is supported on both IPv4 and IPv6, so I think this can be merged.

Reply:
@lzap so we stick to a concept that everything that is documented is supported by default. And if something is not supported - it cannot be found in the documentation. So I think that this statement is fine as it is now. Also, the confirmation that the UEFI HTTP Boot provisioning is supported both on IPv4 and IPv6 makes me calm.
This is the last thing that prevents this PR from being merged. Please elaborate more on what you meant.

Reply:
If Radovan confirmed, then lets's remove the current statement. If it works on both IPv4 and 6 then let's not comment on that.

Reply:
Disregard, the statement is correct.